cmd/rhelemeter-server/README.md

```diff
@@ -0,0 +1,75 @@
+# Rhelemeter
+
+Rhelemeter is a Prometheus remote write forward proxy. It allows RHEL hosts or an intermediate proxy to push metrics to
+a upstream Prometheus (or Prometheus remote write compatible) endpoint.
+
+[rhelemeter-server](https://github.com/openshift/telemeter/tree/master/cmd/rhelemeter-server) needs to receive
+and send metrics across security boundaries, and thus performs additional authorization and data integrity checks as required.
+
+Architecture
+```mermaid
+C4Deployment
+    title rhelemeter-server
+
+    Deployment_Node(rhel-os, "RHEL OS VM", ""){
+        Container(rhel, "Prometheus Remote Write Client", "", "Writes host OS based metrics.")
+    }
+
+    Deployment_Node(proxy, "Intermediate Proxy", ""){
+        Container(remote-write-proxy, "mTLS auth proxy", "", "Validates client certificate and forwards contents as header")
+    }
+        
+    Deployment_Node(rhelemeter, "rhelemeter-server", "Running in Red Hat infrastructure"){
+        Container(rhelemeter-server, "rhelemeter-server", "", "Validates sender and request contents and forwards upstream or rejects.")
+    }
+        
+    Deployment_Node(observatorium, "observatorium", ""){
```

Comment:
We could also name the instance here `observatorium-mst`

Reply:
As above. This describes some "observatorium" backend but I'm not convinced we need to be more detailed in this repo.